index.html

```diff
@@ -376,6 +377,18 @@ <h2>
           The Gamepad's controls have been mapped to the <a href=
           "#remapping">Standard Gamepad layout</a>.
         </dd>
+        <dt>
+          <dfn>xr-standard</dfn>
+        </dt>
+        <dd>
+          The Gamepad's controls have been mapped to the <a href=
+          "https://immersive-web.github.io/webxr-gamepads-module/#xr-standard-gamepad-mapping">
```

Comment:
We'd be better to use /TR/ reference than linking to a draft at github.io.

Reply:
Should happen automatically once the definition is exported and this document is republished.